src/modules/token/fungible/components/SendForm/SendForm.js

```diff
@@ -115,6 +129,22 @@ const SendForm = (props) => {
     return areFieldsValid && isTokenValid;
   }, [amount, recipient, reference, recipientChain, sendingChain, token]);
 
+  useEffect(() => {
+    if (!isLoadingFeeBalance && !tokenBalanceQuery.isLoading) {
```

Comment:
Can we move  `const hasTokenWithBalance` calculation here (lines 89-91). Inside this if statement since this is the only part of the component which uses that logic.